### PR TITLE
Revise dc_ohmic_loss and dc_ohms_from_percent docstrings

### LINKED ISSUE #1601: Inappropriate reference for the function "dc_ohmic_losses"
**Describe the bug**
The function ```dc_ohmic_losses``` consists of one line:
```
return resistance * current * current
```
which is clearly an elementary electrical calculation.

The pvlib documentation provides a reference to the PVsyst documentation and organizes this function under the PVsyst banner in the documentation.

I think this is a bit strange, and could give the impression that this is something invented by/for PVsyst.

The same could be said about the function ```dc_ohms_from_percent```.

**Desired behavior**
It would be preferable to have basic functions like this without such specific attributions/associations.  A reference, if judged necessary, could be to a simple textbook.  It could be useful to provide the information that PVsyst uses these elementary equations, which could be mentioned in the Notes section perhaps--ideally with information about other software too.

### PR DESCRIPTION
**Pull request by cwhanse**
- [X] Closes #1601
 - [X] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - ~~[ ] Tests added~~
 - ~~[ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.~~
 - [x] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [X] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [X] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

#1601 points out that the calculations are elementary and referencing the PVsyst help file isn't appropriate. Being elementary, and completely explained in the docstring, obviates the need for a reference IMO.

The docstring for `dc_ohms_from_percent` was further edited to fix a few typos, adjust to recent style (explanatory Notes section), and clarify module vs. array parameters.

### CODE REVIEW

File: pvlib/pvsystem.py [hunk 1]
```diff
@@ -2872,45 +2872,47 @@ def dc_ohms_from_percent(vmp_ref, imp_ref, dc_ohmic_percent,
                          modules_per_string=1,
                          strings=1):
     """
```
Comment (RDaxini) [inline pvlib/pvsystem.py:2874]: `\times` is not rendering correctly in the docs. Using a raw string for the docstring should fix that. Same goes for `dc_ohmic_losses` but I can't edit that line
```suggestion
    r"""
```

File: pvlib/pvsystem.py [hunk 2]
```diff
@@ -2872,45 +2872,47 @@ def dc_ohms_from_percent(vmp_ref, imp_ref, dc_ohmic_percent,
...
-    ----------
-    .. [1] PVsyst 7 Help. "Array ohmic wiring loss".
-       https://www.pvsyst.com/help/ohmic_loss.htm
+    Notes
+    -----
+    Equivalent resistance is calculated as:
+
+    .. math::
+
+        Rw = (L_{stc} / 100) \times (Varray / Iarray)
```
Comment (RDaxini) [inline pvlib/pvsystem.py:2907]: how about subscript for "array" and "w"? Using `\frac{}{}` might also render the fractions more nicely too. Just some ideas/suggestions
```suggestion
        R_w = \left(\frac{L_{stc}}{100}\right) \times \left(\frac{
        V_{array}}{I_{array}}\right)
```

File: pvlib/pvsystem.py [hunk 3]
```diff
@@ -2872,45 +2872,47 @@ def dc_ohms_from_percent(vmp_ref, imp_ref, dc_ohmic_percent,
...
+
+    :math:`Rw` is the equivalent resistance in ohms.
+    :math:`Varray` is the array voltage, equal to ``vmp_ref`` times
+    ``modules_per_string``.
+    :math:`Iarray` is the array current, equal to ``imp_ref`` times
+    ``strings``.
```
Comment (RDaxini) [inline pvlib/pvsystem.py:2913]: This is just to be consistent with the suggested change to the equation. If you don't go for that then this suggestion can be discarded.
```suggestion
    :math:`R_w` is the equivalent resistance in ohms.
    :math:`V_{array}` is the array voltage, equal to ``vmp_ref`` times
    ``modules_per_string``.
    :math:`I_{array}` is the array current, equal to ``imp_ref`` times
    ``strings``.
```

File: pvlib/pvsystem.py [hunk 4]
```diff
@@ -2929,23 +2931,30 @@ def dc_ohmic_losses(resistance, current):
...
     pvlib.pvsystem.dc_ohms_from_percent
 
-    References
-    ----------
-    .. [1] PVsyst 7 Help. "Array ohmic wiring loss".
-       https://www.pvsyst.com/help/ohmic_loss.htm
+    Notes
+    -----
+    Ohmic (also termed joule or heat) loss is the power lost due to current
+    flowing through a conductor. Ohmic loss :math:`L` is computed as
```
Comment (RDaxini) [inline pvlib/pvsystem.py:2950]:
```suggestion
    flowing through a conductor. Ohmic loss, :math:`L`, is computed as
```

**Review by RDaxini — APPROVED** [review]

**Comment by cwhanse** [comment]
That was a misclick. I thought I was confirming the email associated with the last commit. @pvlib/pvlib-maintainer can do a follow-on if further edits are desired.

File: pvlib/pvsystem.py [hunk 5]
```diff
@@ -2922,30 +2925,37 @@ def dc_ohms_from_percent(vmp_ref, imp_ref, dc_ohmic_percent,
...
-    .. [1] PVsyst 7 Help. "Array ohmic wiring loss".
-       https://www.pvsyst.com/help/ohmic_loss.htm
+    Notes
+    -----
+    Ohmic (also termed joule or heat) loss is the power lost due to current
+    flowing through a conductor. Ohmic loss, :math:`L`, is computed as
+
+    .. math::
+
+        L = I \times R^2
```
Comment (kandersolar) [inline pvlib/pvsystem.py:2955]: This should be L = I^2 R

File: docs/sphinx/source/whatsnew/v0.11.2.rst [hunk 6]
```diff
@@ -14,6 +14,8 @@ Enhancements
 
 Documentation
 ~~~~~~~~~~~~~
+* Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
+  :py:func:`~pvlib.pvsystem.dc_ohmic_loss` for clarity. (:issue:`1601`, :pull:`2229`)
```
Comment (kandersolar) [inline docs/sphinx/source/whatsnew/v0.11.2.rst:18]: should be `dc_ohmic_losses`, not `dc_ohmic_loss`

**Review by kandersolar — COMMENTED** [review]
A couple edits (I guess GitHub does not allow making direct suggestions after the PR isclosed).

**Pull request merged**